app/src/main/java/grinnell/appdev/edu/lyles/fragments/MenuTabFragment.java

```diff
@@ -0,0 +1,108 @@
+package grinnell.appdev.edu.lyles.fragments;
+
+
+import android.annotation.SuppressLint;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ListView;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayList;
+
+import grinnell.appdev.edu.lyles.MenuTabListAdapter;
+import grinnell.appdev.edu.lyles.R;
+
+import static grinnell.appdev.edu.lyles.MenuViewPagerAdapter.DRINKS_INDEX;
+import static grinnell.appdev.edu.lyles.MenuViewPagerAdapter.HOT_FOOD_INDEX;
+import static grinnell.appdev.edu.lyles.MenuViewPagerAdapter.SNACKS_INDEX;
+
+/**
+ * A fragment to display available drinks to users
+ */
+public class MenuTabFragment extends Fragment {
+
+    public final int position;
+
+    public MenuTabFragment() {
+        this.position = -1;
+    }
+
+    @SuppressLint("ValidFragment")
+    public MenuTabFragment(int position) {
+        this.position = position;
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        View view = inflater.inflate(R.layout.fragment_menutab, container, false);
+        ListView list = (ListView) view.findViewById(R.id.menutab_list);
+
+        try {
+            URL url;
+            switch (this.position) {
+                case HOT_FOOD_INDEX:
+                    url = new URL("http://www.cs.grinnell.edu/~birnbaum/appdev/lyles/hotfood.json");
```

Comment:
+1 to what @PsychicNoodles said. It's important to make these non-res strings constants especially when they will not be modified. For performance reasons, it's advisable to declare variables as `static final` whenever possible